### ISSUE TITLE
Competition

### ISSUE THREAD

**Issue by wsjackson14** [yardsalebusiness/Yardsale-#16]
I really liked your idea and your page looks great, but how will you set yourselves apart from the existing companies already doing this same service? I know this issue has already been raised, but I think it's a very important one and it should be addressed in detail on your page.

**Comment by tylerDavisUno** [comment]
I think that a good way to separate yourself from the competition is to maybe have incentives to using your app. Like after selling 3 things you get a coupon or something like that.

**Comment by wsjackson14** [comment]
I like your thinking, but this same incentive exists in other similar businesses. For example, the Mercari app gives you $5 off if you post something to sell or if you buy a certain product, or things such as that. I just think it would be too difficult to differentiate from the existing competitors.

**Comment by manizha83** [comment]
I would agree with Tyler. Offer something to customers that the competitors don't offer. For example lower cost, or some sort of coupons to attract customers into doing business with you over other companies.

**Comment by AliAlherz** [comment]
I think that a good way to separate yourself from the competition is to maybe have incentives exists in other similar businesses Like after selling a lot of things you get a coupon.
